src/Microsoft.CodeQuality.Analyzers/UnitTests/QualityGuidelines/DoNotCallOverridableMethodsInConstructorsTests.cs

```diff
@@ -286,18 +280,23 @@ abstract class F : System.ComponentModel.Component
 
     protected abstract void Foo();
 }
-";
-            Document document = CreateDocument(source, LanguageNames.CSharp);
-            Project project = document.Project.AddMetadataReference(MetadataReference.CreateFromFile(typeof(System.Web.UI.Control).Assembly.Location));
-            project = project.AddMetadataReference(MetadataReference.CreateFromFile(typeof(System.Windows.Forms.Control).Assembly.Location));
-            DiagnosticAnalyzer analyzer = GetCSharpDiagnosticAnalyzer();
-            GetSortedDiagnostics(analyzer, project.Documents.Single()).Verify(analyzer, GetDefaultPath(LanguageNames.CSharp));
+"
+                    },
+                }
+            }.RunAsync();
         }
 
         [Fact]
-        public void CA2214SpecialInheritanceBasic()
+        public async Task CA2214SpecialInheritanceBasic()
```

Comment:
💡 Can we break this test up into one test for System.Web and a second test for System.Windows.Forms?